SECURITY.md

```diff
@@ -0,0 +1,29 @@
+# Security Policy
+
+## Supported Versions
+
+Use this section to tell people about which versions of your project are
+currently being supported with security updates.
+
+| Version | Supported          |
+| ------- | ------------------ |
+| 2.x   | :x:                |
+| 3.x   | :white_check_mark: |
+| 4.x   | :white_check_mark: |
+
+## Reporting a Vulnerability
+
+At Verdaccio, we consider the security of our systems a top priority. But no matter how much effort we put into system security, there can still be vulnerabilities present. If you've discovered a vulnerability, please follow the guidelines below to report it to our team:
+
+* E-mail your findings to `verdaccio@pm.me`. If the report contains highly sensitive information, please consider encrypting your findings using our {PGP key}.
```

Comment:
@DanielRuf do you know what exactly we should define here `{PGP key}` ?

Reply:
A link to the ASCII encoded public ksy (.asc file) probably.

Reply:
A link to the ASCII encoded public key (.asc file) normally.